### PR TITLE
py2 and py3 operability

### PR DESCRIPTION
**Pull request by stg7**
some small fixes for py3

### CODE REVIEW

File: src/demo.py [hunk 1]
```diff
@@ -1,33 +1,34 @@
 #!/usr/bin/env python
-
+from __future__ import print_function
 import phrasefinder
 
+
```
Comment (mtrenkmann) [inline src/demo.py]: are two empty lines python-style?

File: src/demo.py [hunk 2]
```diff
@@ -1,33 +1,34 @@
...
-            print '%6f' % phrase.score,
+            print("{0:6f}".format(phrase.score), end="")
             for token in phrase.tokens:
-                print '%s_%d' % (token.text, token.tag),
-            print
-        print 'Remaining quota: %d' % result.quota
-    
-    except Exception, e:
-        print 'Some error occurred: %s' % e
+                print('{}_{}'.format(token.text, token.tag), end="")
```
Comment (mtrenkmann) [inline src/demo.py]: My test showed that the output is different. There are no spaces between tokens anymore. Must be caused by the new print statement.

File: src/phrasefinder.py [hunk 3]
```diff
@@ -18,13 +18,15 @@
 Module phrasefinder provides routines for querying the PhraseFinder web service
 at http://phrasefinder.io.
 """
-
-try:
+import sys
+if sys.version_info[0] < 3:  # this check is correct
```
Comment (mtrenkmann) [inline src/phrasefinder.py]: Remove comment.

File: src/phrasefinder.py [hunk 4]
```diff
@@ -127,6 +129,7 @@ def search(query, options = Options()):
     if result.status == Status.Ok:
         result.quota = int(file.info()["X-Quota"])
         for line in file.readlines():
+            line = line.decode('utf-8')  # needed because py3 reads bytes via urllib
```
Comment (mtrenkmann) [inline src/phrasefinder.py]: Remove comment.

File: src/demo.py (hunk 1 above)
Comment (stg7) [inline src/demo.py]: yes based on pep8 convention (for both py versions), but phrasefinder lib is far away from pep8

File: src/demo.py (hunk 2 above)
Comment (stg7) [inline src/demo.py]: ok, py2 print automatically adds an " " if you write this print "text", 

should be fixed

File: src/demo.py [hunk 5]
```diff
@@ -23,7 +23,7 @@ def main():
         for phrase in result.phrases:
             print("{0:6f}".format(phrase.score), end="")
             for token in phrase.tokens:
-                print('{}_{}'.format(token.text, token.tag), end="")
+                print('{}_{}'.format(token.text, token.tag), end=" ")
```
Comment (mtrenkmann) [inline src/demo.py]: The space between the score and the first token is still missing. I think
```py
print(' {}_{}'.format(token.text, token.tag), end="")
```

would fix it. Then it's equal to the C and C++ version.

Comment (stg7) [inline src/demo.py]: ok, so this way

**Pull request merged**